client/html/help_search_posts.tpl

```diff
@@ -107,91 +107,107 @@
             <td>having given aspect ratio (image width / image height)</td>
         </tr>
         <tr>
-            <td><code>image-ar</code></td>
+            <td><code>  image-ar</code></td>
             <td>alias of <code>image-aspect-ratio</code></td>
```

Comment:
To make the page less confusing and emphasize the non-alias entries. Aliases are indented since they are not the main way to address a certain keyword.

Reply:
Do you think I should keep it?

Reply:
I would remove it, and keep stuff like that in a separate PR

Reply:
Also, stuff like that should probably be done using CSS

Reply:
Okay then.